tests/integration/targets/rds_cluster_restore/defaults/main.yml

```diff
@@ -0,0 +1,37 @@
+# defaults file for rds_cluster
+
+# Create cluster
+cluster_id: ansible-test-{{ inventory_hostname | replace('_','-') }}{{ tiny_prefix
+  }}
+username: testrdsusername
+password: test-rds_password
```

Comment:
```suggestion
password: "{{ lookup('password', 'dev/null length=12 chars=ascii_letters,digits') }}"
```

Reply:
This is a code reorganization, I don't think this change is required at this stage. The current static password works as expected and if we were to auto-generate it, I would prefer to do it in another patch.